### ISSUE TITLE
[Accessibility] : Multiple Issue with TagPicker Suggestions Props.

### ISSUE THREAD

**Issue by v-sagsah** [microsoft/fluentui#13090]
**Browser Details: ​**
OS: Windows 10 (Version 1903, Build: 18362.175)
Chromium Edge 81.0.416.41 (Official build) beta (64-bit), 
Google Chrome Version 80.0.3987.149

**Issues:**

1. When suggestions for **‘TagPicker’** are opened & user is navigating to the results, after 3 results Narrator focus is going back to aria-label text and reading it every-time along with Search result.
2. When suggestion drop-down is opened, visually it shows top 3 results but keyboard and screen reader users can navigate to all the results available by pressing the down arrow key. 
3. The focus does not now goes to the _**"Show More 3 Items"**_ after the 3rd item. Focus goes, after the last items in the list, in my case after the 9th item.
4. Screen reader is not reading count of the option after the 3rd Item option in the auto suggestion dropdown.
5. When zoomed to 400%, few options text is not completely visible.
6. Narrator is reading the 1st option, in the Tag Picker 2 times.

All the above 3 issues are reproducible in the below code-pen link :
https://codepen.io/sagsah/full/RwWxxpw

**Expected Result: ​**
1. Narrator/NVDA should not read the place holder information when navigating to the search results present after first 3 items. Narrator has to read the search result name and read the position as 4 of total search results. The focus also should not go back to the aria label text after the 3rd item. The focus should go to _**"Show More 3 Items"**_ after the 3rd item.
2. As visually it is given as top 3 results and screen reader is reading as top 3 results, there should not be more than 3 results in the drop-down and user needs to press show more to load the next 3 items.
3. Screen reader should read the count of the focused option correctly.
4. Every text and control should be visible and accessible when zoomed to 400%.
5. Narrator should not read the 1st option, in the Tag Picker 2 times.

**User Impact: ​**
1. Screen Reader users will get confused if screen reader is reading the place holder text every time.
2. Users with disabilities will not be able to understand if the text is given as 3 results but users can navigate to all the results.
3. Screen Reader users may find it difficult to know the count of the option and total number of options.
4. Low vision users may find it difficult to understand the complete value of the option while navigating through the auto suggestion dropdown.

**Comment by aneeshack4** [comment]
Hi, thanks for reporting these @v-sagsah. I see them all repro on Narrator. I looked at the BasePicker code and it seems that ARIA spec is being followed. Also ran Accessibility Insights and nothing was flagged: 
![image](https://user-images.githubusercontent.com/4161791/81581607-9562f000-9374-11ea-999b-7a11c6b1be0c.png)
Tagging @joschect the component owner - do you know what the cause here could be and a possible workaround or fix? If it's an expensive fix, we likely won't make it at this time as we're focused on larger worker items (see pinned issue) but we'll get to this when we schedule future work on Pickers.

**Comment by v-sagsah** [comment]
@aneeshack4 & @joschect I found one more issue, please find it below:

**Issue:**
Screen reader is not reading count of the option after the 3rd Item option in the auto suggestion dropdown.

**Expected Result: ​**
Screen reader should read the count of the focused option correctly.

I have also updated same above. You can able to repro the same in the below code-pen link :
https://codepen.io/sagsah/full/RwWxxpw

**Comment by v-sagsah** [comment]
@aneeshack4 & @joschect , request you to please update on the same, with any workaround or fix?

**Comment by v-sagsah** [comment]
@aneeshack4  / @joschect , can please provide an update on these ?

**Comment by sagnik17** [comment]
@aneeshack4 / @joschect can you please provide any updates on these ???

**Comment by v-sagsah** [comment]
@aneeshack4 / @joschect , can you please provide an update on these ???

**Comment by ecraig12345** [comment]
@v-sagsah I've asked someone to take a look at the suggestions dropdown issues.

Please open a separate issue about the 400% zoom problems (we can't address it right away but will need to fix it eventually), as well as a separate issue for reading the first selected tag twice (since that's unrelated to the suggestions dropdown).

In general, we prefer that you make a separate issue for each problem rather than lumping them together, because each one will likely need to be triaged and addressed differently. You'll probably also get faster responses for separate issues.

**Comment by joschect** [comment]
@v-sagsah reply inline.

1. When suggestions for ‘TagPicker’ are opened & user is navigating to the results, after 3 results Narrator focus is going back to aria-label text and reading it every-time along with Search result.
* This one will need more investigation. This is likely a narrator problem as we should be following the combobox structure fairly closely. 
2. When suggestion drop-down is opened, visually it shows top 3 results but keyboard and screen reader users can navigate to all the results available by pressing the down arrow key.
* This is expected, it works just like a dropdown does. You can limit the amount of items in the list by restricting the amount returned by onResolveSuggestions. Otherwise the navigation box has a scrollbar because it is populated with all the items.
3. The focus does not now goes to the "Show More 3 Items" after the 3rd item. Focus goes, after the last items in the list, in my case after the 9th item.
* This is the same response as 2. The entire list returned by onResolveSuggestions will appear in the suggestions dropdown. It's up to you to restrict the results returned. You can see how this is done in the [People Picker](https://codepen.io/slimemold/pen/XWmOVpZ?editable=true%3Dhttps%3A%2F%2Fdeveloper.microsoft.com%2Fen-us%2Ffluentui) which uses the same base control.
4. Screen reader is not reading count of the option after the 3rd Item option in the auto suggestion dropdown.
* Like 1 this will require more investigation. It's likely that this is a screenreader/narrator issues since all of the aria is appropriate.
5. When zoomed to 400%, few options text is not completely visible.
* @ecraig12345 's response on this is correct.
6. Narrator is reading the 1st option, in the Tag Picker 2 times.
* @ecraig12345 's response on this is also good.

**Comment by v-sagsah** [comment]
Thank you @ecraig12345 & @joschect for your comments. As mentioned I have created separate issue individual issues for each of them, so closing this one.
